pkg/handler/v2/external_cluster/external_cluster.go

```diff
@@ -1008,8 +1017,9 @@ type body struct {
 	// Name is human readable name for the external cluster
 	Name string `json:"name"`
 	// Kubeconfig Base64 encoded kubeconfig
-	Kubeconfig string                          `json:"kubeconfig,omitempty"`
-	Cloud      *apiv2.ExternalClusterCloudSpec `json:"cloud,omitempty"`
+	Kubeconfig    string                          `json:"kubeconfig,omitempty"`
+	SSHPrivateKey string                          `json:"ssh-privatekey,omitempty"`
```

Comment:
change to `sshKey`

Reply:
earlier in KKP code, we mentioned if the key is public https://github.com/kubermatic/kubermatic/blob/master/pkg/api/v1/types.go#L468.
WDYT?

Reply:
can stay SSHPrivateKey but please change the swagger to `sshPrivateKey`